SCClassLibrary/JITLib/ProxySpace/wrapForNodeProxy.sc

```diff
@@ -307,7 +307,8 @@
 						XOut.ar(out, env, SynthDef.wrap(func, nil, [In.ar(out, proxy.numChannels)]))
 					} {
 						env = ctl * EnvGate(i_level: 0, doneAction:2, curve:\lin);
-						XOut.kr(out, env, SynthDef.wrap(func, nil, [In.kr(out, proxy.numChannels)]))				};
+						XOut.kr(out, env, SynthDef.wrap(func, nil, [In.kr(out, proxy.numChannels)]))
```

Comment:
Are you sure that this shouldn't be a `wetamp * In.kr(out, proxy.numChannels)` ?

Reply:
ah I see, it is already multiplying above.

Reply:
For reference: this is just a formatting change.

Reply:
> For reference: this is just a formatting change.

@telephon - which change is just a formatting change? 
overall, the commit fixes a bug which made \filterIn behave like \filter; 
you can test it with the examples given to see the different behavior.